theming.js

```diff
@@ -17,30 +17,40 @@ const Dash = imports.ui.dash;
 const DND = imports.ui.dnd;
 const IconGrid = imports.ui.iconGrid;
 const Main = imports.ui.main;
+const Panel = Main.panel;
 const PopupMenu = imports.ui.popupMenu;
 const Tweener = imports.ui.tweener;
 const Util = imports.misc.util;
 const Workspace = imports.ui.workspace;
 
 const Me = imports.misc.extensionUtils.getCurrentExtension();
+const Dock = Me.imports.docking;
 const Utils = Me.imports.utils;
 
+const TransparencyMode = {
+    default:  0,
+    fixed:    1,
+    adaptive: 2,
+    dynamic:  3
+};
+
```

Comment:
Purely style: we have other "ENUMs" with capitalized values.

Reply:
Also, we need some comment defining what those means.

Reply:
Addresses here [b52f68f50cb68befc4c258093590ee74ff47192b].